### DISCUSSION TITLE
Unlocks my car when Home Assistant is restarted

### DISCUSSION THREAD

**Discussion by ChipMIK1** [alandtse/tesla#147] [Q&A] [answered]
Hi 

Do you guys also experience your car unlocks when Home Assistant is restarted? 

I have had some strange behaviour for quite some time with my car suddenly being unlocked but found out today that this happends when i restart Home Assistant.

Is there a fix for this somehow if any other also experience this?

**Comment by alandtse** [comment, marked as answer]
This component does not issue any commands on restart. We only pull into from Tesla. If you added any scripts in HA that unlock the car, those could be triggering.

Alternatively you have not fully secured your HA and someone else is accessing your Tesla.

**Reply by ChipMIK1** [reply]
Now why didnt i think about scripts/automation. I have 2 automations in relation to unlocking the car. 1 for an IKEA Zigbee button & on for an RF keyring remote where it goes through a Sonoff RF unit thats flashed with ESPHome. From your input i just found out that when i disable the RF automation the car no longer unlocks on HA restart. 

Thank you very much for reply :-)
